rpm-testing/install.sh

```diff
@@ -0,0 +1,164 @@
+[archivematica]
+name=archivematica
+baseurl=https://packages.archivematica.org/1.7.x/centos
+gpgcheck=1
+gpgkey=https://packages.archivematica.org/1.7.x/key.asc
+enabled=1
+
+[archivematica-extras]
+name=archivematica-extras
+baseurl=https://packages.archivematica.org/1.7.x/centos-extras
+gpgcheck=1
+gpgkey=https://packages.archivematica.org/1.7.x/key.asc
+enabled=1
+EOF'
+fi
+
+sudo -u root yum update -y
+sudo -u root yum install -y epel-release policycoreutils-python
+
+
+#
+# SELinux tweaks
+#
+
+if getenforce; then
+    sudo semanage port -m -t http_port_t -p tcp 81
+    sudo semanage port -a -t http_port_t -p tcp 8001
+    sudo setsebool -P httpd_can_network_connect_db=1
+    sudo setsebool -P httpd_can_network_connect=1
+    sudo setsebool -P httpd_setrlimit 1
+fi
+
+
+#
+# Install MariaDB and Gearman
+#
+
+sudo -u root yum install -y mariadb-server gearmand
+sudo -u root systemctl enable mariadb
+sudo -u root systemctl start mariadb
+sudo -u root systemctl enable gearmand
+sudo -u root systemctl start gearmand
+
+
+if [ "${search_enabled}" == "true" ] ; then
+    sudo -u root rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch
+    sudo -u root bash -c 'cat << EOF > /etc/yum.repos.d/elasticsearch.repo
+[elasticsearch-1.7]
+name=Elasticsearch repository for 1.7 packages
+baseurl=https://packages.elastic.co/elasticsearch/1.7/centos
+gpgcheck=1
+gpgkey=https://packages.elastic.co/GPG-KEY-elasticsearch
+enabled=1
+EOF'
+  sudo -u root yum install -y java-1.8.0-openjdk-headless elasticsearch
+  sudo -u root systemctl enable elasticsearch
+  sudo -u root systemctl start elasticsearch
+fi
+
+#
+# Archivematica Storage Service
+#
+
+sudo -u root yum install -y python-pip archivematica-storage-service
+sudo -u archivematica bash -c " \
+  set -a -e -x
+  source /etc/sysconfig/archivematica-storage-service
+  cd /usr/lib/archivematica/storage-service
+  /usr/share/python/archivematica-storage-service/bin/python manage.py migrate
+";
+
+sudo -u root systemctl enable archivematica-storage-service
+sudo -u root systemctl start archivematica-storage-service
+sudo -u root systemctl enable nginx
+sudo -u root systemctl start nginx
+sudo -u root systemctl enable rngd
+sudo -u root systemctl start rngd
+
+
+#
+# Dashboard and MCPServer
+#
+
+sudo -u root yum install -y archivematica-common archivematica-mcp-server archivematica-dashboard
+
+sudo -H -u root mysql -hlocalhost -uroot -e "DROP DATABASE IF EXISTS MCP; CREATE DATABASE MCP CHARACTER SET utf8 COLLATE utf8_unicode_ci;"
+sudo -H -u root mysql -hlocalhost -uroot -e "CREATE USER 'archivematica'@'localhost' IDENTIFIED BY 'demo';"
+sudo -H -u root mysql -hlocalhost -uroot -e "GRANT ALL ON MCP.* TO 'archivematica'@'localhost';"
+
+sudo -u archivematica bash -c " \
+  set -a -e -x
+  source /etc/sysconfig/archivematica-dashboard
+  cd /usr/share/archivematica/dashboard
+  /usr/share/python/archivematica-dashboard/bin/python manage.py migrate --noinput
```

Comment:
After merging https://github.com/artefactual-labs/am-packbuild/pull/130, this needs to be updated